MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+include CHANGELOG.* CONTRIBUTORS.* LICENSE tox.ini
+graft src
+graft test
+include test_script.py
```

Comment:
@jwodder why include tests in the sdist?

Reply:
So that repackagers who build off the sdist can run the tests.